mllib/src/main/scala/org/apache/spark/mllib/api/python/PythonMLLibAPI.scala

```diff
@@ -465,6 +465,36 @@ private[python] class PythonMLLibAPI extends Serializable {
   }
 
   /**
+   * Java stub for Python mllib LDA.run()
+   */
+  def trainLDAModel(
+      data: JavaRDD[LabeledPoint],
```

Comment:
Umm, it seems that we can't deal with `JavaRDD[(Long, Vector)]` in the parameter of `trainLDAModel`. After all, I got the error message as follows, when I run the test. Tuple of Python was recognized as `java.lang.Object` in Java.

```
java.lang.ClassCastException: [Ljava.lang.Object; cannot be cast to scala.Tuple2
        at org.apache.spark.mllib.api.python.PythonMLLibAPI$$anonfun$6.apply(PythonMLLibAPI.scala:489)
        at scala.collection.Iterator$$anon$11.next(Iterator.scala:328)
        at scala.collection.Iterator$$anon$11.next(Iterator.scala:328)
...
```

According to `trainFPGrowthModel` and `trainWord2Vec`, dealing with input data as array would be better. What do you think about this implementation?

https://github.com/yu-iskw/spark/commit/cdb2cf167097dda4f15eb690b7a73cadabdd3313